database/sqlite/sqlite_aclk.c

```diff
@@ -722,19 +711,20 @@ void sql_check_aclk_table_list(struct aclk_database_worker_config *wc)
     return;
 }
 
-void aclk_data_rotated(RRDHOST *host)
+void aclk_data_rotated(void)
 {
-    UNUSED(host);
+#if ENABLE_NEW_CLOUD_PROTOCOL
```

Comment:
This one `#if` too ?

Reply:
yes made the changes already (won't push just yet to see if we have other changes)